crates/ruff_python_formatter/README.md

```diff
@@ -94,18 +94,30 @@ write!(f, [if_group_breaks(&text(","))])
 If you need avoid second mutable borrows with a builder, you can use `format_with(|f| { ... })` as
 a formattable element similar to `text()` or `group()`.
 
-The generic comment formatting in `FormatNodeRule` handles comments correctly for most nodes, e.g.
-preceding and end-of-line comments depending on the node range. Sometimes however, you may have
-dangling comments that are not before or after a node but inside of it, e.g.
+## Comments
+
+Comments can either be own line or end-of-line and can be marked as `Leading`, `Trailing` and `Dangling`.
```

Comment:
Unfortunately, we don't host our rust documentation anywhere. We could otherwise link to the rust documentation.

Reply:
Would be nice to put this somewhere under https://beta.ruff.rs/docs/, the rustdoc is static files only